CONTRIBUTING.md

```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
 
 # For Project Maintainers
 
-## Releasing to NPM
+## Releasing to npm
+
+Releasing to npm consists of the following phases:
 
-Releasing to NPM consists of 3 phases:
 1. Source Code: mark a release.
     1. Bump `package.json` version following the SEMVER rules and send a PR titled `'chore: mark version vXXX.YYY.ZZZ'` ([example](https://github.com/GoogleChrome/puppeteer/commit/808bf8e5582482a1d849ff22a51e52024810905c)).
     2. Make sure the PR passes **all checks**.
         - **WHY**: there are linters in place that help to avoid unnecessary errors, e.g. [like this](https://github.com/GoogleChrome/puppeteer/pull/2446)
     3. Merge the PR.
     4. Once merged, publish release notes using the "create new tag" option.
```

Comment:
> the "create new tag" option

What is this referring to exactly? On https://github.com/GoogleChrome/puppeteer/releases I see a "Draft a new release" button -- is that what is meant? 

Is there some tool or `git log`-based command-line script you're using for the release notes ([example](https://github.com/GoogleChrome/puppeteer/releases/tag/v1.19.0))?

Reply:
https://github.com/GoogleChrome/puppeteer/pull/4940